watchtower/wtdb/tower.go

```diff
@@ -15,12 +15,31 @@ var (
 	ErrTowerNotFound = errors.New("tower not found")
 )
 
+// TowerID is a unique 64-bit identifier allocated to each unique watchtower.
+// This allows the client to conserve on-disk space by not needing to always
+// reference towers by their pubkey.
+type TowerID uint64
```

Comment:
Yes you're right, in the initial iteration the acks will dwarf this savings. my plan in the future is to move to a persistent RLE encoding for a channel's `CommitHeights`, since we expect most of them to be contiguous. This should make the disparity significantly less.

As we move to support multiple towers, we will also want an efficient way to reference sets of towers holding a given ranges of updates, so the savings from `TowerID` will be more apparent then as well

In light of this, do you think it's worth adding later or keeping?

Reply:
In general I am cautious of future proofing, because it may not happen (or happen in a different way). But if you think it is pretty sure we'll regret not adding the `TowerID` now, you should leave it in.

Reply:
opted to leave it in, since overall the code diff is pretty small and i believe it will offer space savings when tracking backups in the multi-tower case :)